internal/pkg/cli/env_deploy.go

```diff
@@ -123,6 +136,15 @@ func (o *deployEnvOpts) Execute() error {
 	if err != nil {
 		return err
 	}
+	if mft.CDNConfig.CDNEnabled() && mft.HTTPConfig.Public.Certificates == nil && o.targetApp.Domain != "" {
```

Comment:
Note that we might need a dynamic manifest struct for env manifest (just like the one we do for the workload manifest) so as to resolve dynamic values by making API calls. maybe add a TODO?

Reply:
I don't grasp what this means exactly, can you write a TODO comment I can add there since rn I don't have the time to research what that means?

Reply:
Oh yeah just a

```
// TODO: Put into a new DynamicEnvironmentManifest struct in "manifest" package
```